config.toml

```diff
@@ -0,0 +1,85 @@
+[LOADED_NETWORK]

```

Comment:
Since "LOADED_NETWORK" is the key, it can be dropped from the variable names nested under it.  I suggest "path", "timeperiods", and "columns"